src/audio/kpb.c

```diff
@@ -1741,6 +1835,8 @@ static void kpb_copy_samples(struct comp_buffer __sparse_cache *sink,
 #endif
 #if CONFIG_FORMAT_S24LE || CONFIG_FORMAT_S32LE
 	case 24:
+		samples = size / ((sample_width / 8) * channels);
+		kpb_copy_24b_in_32b(istream, 0, ostream, 0, samples);
```

Comment:
ditto consider adding fallthrough annotation.

Reply:
...also unless a `break` is missing here too